src/benchmarks/gc/GC.Infrastructure/GC.Infrastructure/Commands/ASPNetBenchmarks/AspNetBenchmarksCommand.cs

```diff
@@ -132,27 +139,32 @@ public static AspNetBenchmarkResults RunASPNetBenchmarks(ASPNetBenchmarksConfigu
                         }
                     }
 
-                    string outputDetails = output.ToString();
-                    File.WriteAllText(Path.Combine(outputPath, $"{GetKey(c.Key, run.Key)}.log"), "Output: \n" + outputDetails + "\n Errors: \n" + error.ToString());
+                    // For the case where the output file doesn't exist implies that was an issue connecting to the asp.net machines or error number 1.
+                    // This case also applies for incorrect crank arguments or error number 2.
+                    // Move the standard out to the standard error as the process failed.
```

Comment:
style - move comment into the `else`?  I personally find this a bit confusing because it makes the `if` looks like it doesn't have an `else`